tests/Functional/tests/quote3/quote3.rst

```diff
@@ -1,3 +1,5 @@
 .. image:: /test.jpg
 
+Here is some text.
+
     This should be a quote, right?
```

Comment:
I discovered (by testing in Sphinx) that the previous file was illegal: you an't have an image directive with content in it :)